manager/src/main/java/com/redhat/service/bridge/manager/workers/resources/ConnectorWorker.java

```diff
@@ -0,0 +1,251 @@
+package com.redhat.service.bridge.manager.workers.resources;
+
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.util.Objects;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.openshift.cloud.api.connector.models.Connector;
+import com.openshift.cloud.api.connector.models.ConnectorRequest;
+import com.openshift.cloud.api.connector.models.ConnectorState;
+import com.openshift.cloud.api.connector.models.ConnectorStatusStatus;
+import com.openshift.cloud.api.connector.models.DeploymentLocation;
+import com.openshift.cloud.api.connector.models.KafkaConnectionSettings;
+import com.openshift.cloud.api.connector.models.ServiceAccount;
+import com.redhat.service.bridge.infra.models.dto.ManagedResourceStatus;
+import com.redhat.service.bridge.manager.RhoasService;
+import com.redhat.service.bridge.manager.connectors.ConnectorsApiClient;
+import com.redhat.service.bridge.manager.dao.ConnectorsDAO;
+import com.redhat.service.bridge.manager.models.ConnectorEntity;
+import com.redhat.service.bridge.rhoas.RhoasTopicAccessType;
+
+import io.quarkus.hibernate.orm.panache.PanacheRepositoryBase;
+
+@ApplicationScoped
+public class ConnectorWorker extends AbstractWorker<ConnectorEntity> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ConnectorWorker.class);
+
+    public static final String KAFKA_ID_IGNORED = "kafkaId-ignored";
+
+    @ConfigProperty(name = "managed-connectors.cluster.id")
+    String mcClusterId;
+
+    @ConfigProperty(name = "managed-connectors.kafka.bootstrap.servers")
+    String kafkaBootstrapServer;
+
+    @ConfigProperty(name = "managed-connectors.kafka.client.id")
+    String serviceAccountId;
+
+    @ConfigProperty(name = "managed-connectors.kafka.client.secret")
+    String serviceAccountSecret;
+
+    @Inject
+    ConnectorsDAO connectorsDAO;
+
+    @Inject
+    RhoasService rhoasService;
+
+    @Inject
+    ConnectorsApiClient connectorsApi;
+
+    @Override
+    protected PanacheRepositoryBase<ConnectorEntity, String> getDao() {
+        return connectorsDAO;
+    }
+
+    @Override
+    protected ConnectorEntity runCreateOfDependencies(ConnectorEntity connectorEntity) {
+        // This is idempotent as it gets overridden later depending on actual state
+        connectorEntity = setStatus(connectorEntity, ManagedResourceStatus.PROVISIONING);
+
+        // Step 1 - Create Kafka Topic
+        info(LOGGER,
```

Comment:
I'd do only one log `info` at the beginning of the processing and then use `debug` if it makes sense to you. Same for the other workers/methods

Reply:
Done. I also removed my "helpful" logging methods.. Their purpose was to avoid repeating calls to `if(LOGGER.isXYZEnabled())` however the whole point of the check was to avoid the `String` concatenation.. but it happened regardless when constructing the parameters to my "helpful" methods.